filters/auth/oidc.go

```diff
@@ -471,17 +479,23 @@ func (f *tokenOidcFilter) Request(ctx filters.FilterContext) {
 		allowed = f.validateAllClaims(container.Claims)
 		log.Debugf("validateAllClaims: %v", allowed)
 		sub = container.Subject
+		oidcInfo = container
 	default:
 		unauthorized(ctx, "unknown", invalidFilter, r.Host)
 		return
 	}
 
 	if !allowed {
-		unauthorized(ctx, "insufficient permissions :"+sub, invalidClaim, r.Host)
+		unauthorized(ctx, sub, invalidClaim, r.Host)
```

Comment:
Any reason why not returning a "nicer" string?
`sub` in our cases is a string that looks like a uuid...

Reply:
Mainly because the other uses of `sub` it is used to indicate the subject/username of the failed request. I'm trying to keep it consistent.